src/xpcc/architecture/platform/driver/can/stm32/can.hpp.in

```diff
@@ -77,6 +77,10 @@ public:
 	static const TypeId::Can{{ id }}Rx Rx;
 	/// TypeId used to connect GPIO pins to this peripheral's tx.
 	static const TypeId::Can{{ id }}Tx Tx;
+public:
+	/// Expose jinja template parameters to be checked by e.g. drivers or application
+	static constexpr uint8_t rxBufferSize = parameters.rx_buffer;
+	static constexpr uint8_t txBufferSize = parameters.tx_buffer;
```

Comment:
Missing braces

Reply:
This was a test for the CI ;-)
Travis caught this, CircleCI's integration to GitHub does not yet run smooth. I'll have a look.